platform/jewel/detekt-plugin/src/test/resources/TitleBarStyling-forTest.kt

```diff
@@ -0,0 +1,253 @@
+// Copyright 2000-2025 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
You can ignore this file, I just copied it in here to use it as a "realistic use case" in rule tests